controllers/utils/utils_test.go

```
@@ -0,0 +1,107 @@
+
+	AfterEach(func() {
+		// Restore original env var
+		if originalEnvValue != "" {
+			Expect(os.Setenv("APPLICATIONS_NAMESPACE", originalEnvValue)).To(Succeed())
+		} else {
+			Expect(os.Unsetenv("APPLICATIONS_NAMESPACE")).To(Succeed())
+		}
+
+		// Clean up temp directory if created
+		if tempDir != "" {
+			os.RemoveAll(tempDir)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)